### ISSUE TITLE
Using the theme on Network A

### ISSUE THREAD

**Issue by thriuin** [open-data/ckanext-wet-boew#99]
CKAN may very well be used in a network location where access to CDN locations like ajax.googleapis.com may not be available. We should have an option to be able to load jquery.js from a local server.

**Comment by LaurentGoderre** [comment]
Fair enough. Although I am wondering if WET should ship with it's versions of jQuery so we don't have to

**Comment by thriuin** [comment]
In the case of intranet sites like ours, it would be a requirement. There's probably benefits to having it hosted elsewhere, but it would be good to at least have it as an option and the CDN could be a default. What do you think?

**Comment by LaurentGoderre** [comment]
Yeah I agree but I think WET should do it for us in the packaging, better than us trying to manage versions etc.

**Comment by LaurentGoderre** [comment]
FYI https://github.com/wet-boew/wet-boew/pull/7037

**Comment by thriuin** [comment]
For sure, even better. Thanks!
